simple_qa_rnn/.gitignore

```diff
@@ -1,4 +1,8 @@
-data/
+data/*
```

Comment:
Should all data go into `castorini/data`? Maybe at least the common data like common datasets and embeddings (GloVe) for sure. I made #16 that will put GloVe there. So we probably should have these in the gitignore for simple_qa_rnn.